### PR TITLE
Add maker cli

### LINKED ISSUE #217: Parse command line arguments to configure maker
similar to how we have done it in itchysats using https://docs.rs/clap/latest/clap/

### PR DESCRIPTION
**Pull request by bonomat**
Once this is in we can do #228 

Resolves #217

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *
 !target/release/maker
+!target/x86_64-unknown-linux-musl/release/maker
```
Comment (bonomat) [inline .dockerignore:3]: was not on purpose

Comment (klochowicz) [reply .dockerignore]: what Docker container we will run it in? with `musl`, everything libc should be statically linked, so we could use a minimal one I guess ;)

**Review by klochowicz — APPROVED** [review]
really cool!

File: .dockerignore (hunk 1 above)
Comment (bonomat) [reply .dockerignore]: I just needed this to be able to cross-compile locally :) We are using the standard linux target in our container

**Pull request merged**